.github/workflows/canary.yml

```diff
@@ -35,8 +35,10 @@ jobs:
         with:
           path: |
             ${{ github.workspace }}/packages/*/*.tgz
+            ${{ github.workspace }}/dev-packages/*/*.tgz
```

Comment:
When are we emitting tarballs in the dev-packages?

Reply:
probably not at all right now, we can remove this I guess!